File: 58. Length of Last Word/58.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        valid = False
        answer = 0
        while True:

            if s[-1] != " ":
                if valid == False:
                    valid = True
                answer += 1
                s = s[:-1]

                if len(s) == 0:
                    return answer
            

            elif s[-1] == " ":
                if valid == True:
                    return answer
                s = s[:-1]
            
            
            if len(s) == 0:
                return answer


S = Solution()
print(S.lengthOfLastWord("Hello World"))

File: 58. Length of Last Word/58.cs
public class Solution {
    public int LengthOfLastWord(string s) {
        int lenght = 0;
        int i = s.Count() - 1;

        while(i >=0 && s[i] == ' ')
        {
            i--;
        }
        while(i>=0 && s[i] != ' ')
        {
            lenght++;
            i--;
        }
        return lenght;
    }
}